### DISCUSSION TITLE
Version 5.0.3

### DISCUSSION THREAD

**Discussion by jamonholmgren** [mobxjs/mobx-state-tree#1787] [General]
This patch-level release has a few small updates:

* Removed a couple MobX TypeScript properties that have already been removed in MobX in #1666 by @Strandinator
* Fixes array reconciliation in snapshotProcessor in #1784 by @adamkovalsky, fixing #1776
* Updates devDependencies across the project in #1786 by @jamonholmgren 
* Some documentation updates as well

If there are any problems with this release, please file an issue or leave comments in the attached Discussion! Thanks to everyone who contributed!

<hr /><em>This discussion was created from the release <a href='https://github.com/mobxjs/mobx-state-tree/releases/tag/v5.0.3'>Version 5.0.3</a>.</em>

**Comment by adamkovalsky** [comment]
Unfortunately some more issues related to `snapshotProcessor` have emerged as a result of #1784. I have introduced PR #1792 to address these issues, and hopefully another patch can be released to resolve the regression.

**Reply by jamonholmgren** [reply]
Pushed new release with this fix; thank you! https://github.com/mobxjs/mobx-state-tree/releases/tag/v5.0.4
